### PR TITLE
Add Set Enabled distribute screen

### PR DESCRIPTION
**Pull request by erychagov**

### CODE REVIEW

**Comment by erychagov** [comment]
@clpolet Thanks! I skipped an one place. Fixed!

File: SasquatchSwift/SasquatchSwift/ViewControllers/MainViewController/ViewController.swift [hunk 1]
```diff
@@ -64,7 +67,19 @@ class ViewController: UIViewController {
...
+    enum MSDistributeCases : Int {
+        case SetEnabled
+        var cellSetting : (title:String, type:MSCellType) {
+            switch self {
+            case .SetEnabled:
+                return ("Set Enabled", .Switch)
+            }
+        }
+
+        static let allCases = [MSAnalyticsCases.SetEnabled]
```
Comment (TroubleMakerBen) [inline SasquatchSwift/SasquatchSwift/ViewControllers/MainViewController/ViewController.swift]: `MSAnalyticsCases`?

File: SasquatchSwift/SasquatchSwift/ViewControllers/MainViewController/ViewController.swift [hunk 2]
```diff
@@ -130,48 +186,57 @@ extension ViewController : UITableViewDataSource {
...
             switch MSAnalyticsCases.allCases[indexPath.row] {
             //Track Event
             case .SetEnabled:
                 //Enable/Disable MSAnalytics
                 MSAnalytics.setEnabled(!MSAnalytics.isEnabled())
                 tableView.reloadRows(at: [indexPath], with: .automatic)
-                
+                break
             case .TrackEvent:
                 //Track event with name only
```
Comment (TroubleMakerBen) [inline SasquatchSwift/SasquatchSwift/ViewControllers/MainViewController/ViewController.swift]: Please add a newline here

File: SasquatchSwift/SasquatchSwift/ViewControllers/MainViewController/ViewController.swift [hunk 3]
```diff
@@ -130,48 +186,57 @@ extension ViewController : UITableViewDataSource {
...
+        case .Crashes:
             switch MSCrashesCases.allCases[indexPath.row] {
             case .SetEnabled:
                 //Enable/Disable MSCrashes
                 MSCrashes.setEnabled(!MSCrashes.isEnabled())
                 tableView.reloadRows(at: [indexPath], with: .automatic)
-
+                break
             case .GenerateTestCrash:
                 //Test either debugger attached
```
Comment (TroubleMakerBen) [inline SasquatchSwift/SasquatchSwift/ViewControllers/MainViewController/ViewController.swift]: Add newline above comment

**Comment by erychagov** [comment]
@TroubleMakerBen @clpolet @jaelim-ms 
Why we have different indents for SasquatchObjC (2 spaces) and SasquatchSwift (4 spaces)? Let's make it the same.

**Comment by clpolet** [comment]
@erychagov You are right it should be **2** spaces both.

**Pull request merged**